services/mirror/mirror_push.go

```diff
@@ -141,7 +146,7 @@ func runPushSync(ctx context.Context, m *repo_model.PushMirror) error {
 		if setting.LFS.StartServer {
 			log.Trace("SyncMirrors [repo: %-v]: syncing LFS objects...", m.Repo)
 
-			gitRepo, err := git.OpenRepository(ctx, path)
+			gitRepo, err := gitrepo.OpenRepository(ctx, repo)
```

Comment:
Not sure about this, why not `gitrepo.OpenWikiRepository` if `isWiki` is true?

Reply:
Good catch. Fixed.